### PR TITLE
JSday Maceio recap

### PR DESCRIPTION
**Pull request by juarezpaf**
### Added

- Add JSday Maceio 2015 post
- Add me as part of the `team.yml` file

### CODE REVIEW

**Comment by juarezpaf** [comment]
@hannesvdvreken @Anahkiasen could you guys please provide a feedback on this one. Going to add more pictures tomorrow. :smile:

File: _data/team.yml [hunk 1]
```diff
@@ -22,10 +14,22 @@ dimitris:
...
+  name: Jonas De Hovre
+  twitter: jonasdehovre
+
+juarezfilho:
+  name: Juarez Filho
+  twitter: juarezpaf
+
+tony:
+  name: Tony Messias
+  twitter: tony0x01
```
Comment (hannesvdvreken) [inline _data/team.yml]: Did you sort it? :open_mouth: :heart:

File: _posts/2015-09-21-jsdaymcz-2015.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+title: "JSday Maceio 2015 recap"
+date: 2015-09-21
+author: juarezfilho
+categories: ["javascript", "conference", "jsday", "jsdaymcz"]
+excerpt: "A recap on JSDay Maceio 2015, through the eyes of our employees"
+---
+
+# JSday Maceio 2015
+
+We have tons of conferences in Brazil related to web technologies, BrazilJS is far the most incredible one, and it relies heavily on JavaScript, yearly thousand of JavaScript enthusiasts and developers go to POA to have this amazing experience. In a few months ago I saw a conference called *JSday* and I thought it should be good to bring this experience to other cities, the first one was held in Feira de Santana/BA, I talked with the organizer and he got very excited to see JSday going to other cities.
```
Comment (hannesvdvreken) [inline _posts/2015-09-21-jsdaymcz-2015.md]: We have tons of conferences in Brazil related to web technologies.*\<end of sentence\>*

BrazilJS is far the most incredible one, and it relies heavily on JavaScript.*\<end of sentence\>* **Y**early thousand**s** of JavaScript...

~~In~~ **A** few months ago I saw a conference called...

to bring this experience to other cities.*\<end of sentence\>* **T**he first one was held in Feira de Santana/BA.*\<end of sentence\>*

File: _posts/2015-09-21-jsdaymcz-2015.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+> [JSday Maceió](http://jsday.com.br) is an immersive event focused in JavaScript and everything related to the most used language in the world can offer.
```
Comment (hannesvdvreken) [inline _posts/2015-09-21-jsdaymcz-2015.md]: and everything related to **what** the most used language in the world can offer.

File: _posts/2015-09-21-jsdaymcz-2015.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+Based on this quotation we tried to bring relevant topics to the stage and we accomplished that talking about AngularJS, Google Maps API, lots of NodeJS, Wearable JavaScript, Electron, NodeBots, Firefox OS and Ionic SDK.
+
+We had a great line-up bringing a variety of topics. Speakers from well trustworthy companies around the world like ThoughtWorks, CESAR, madewithlove like Wilson Mendes,  Bruno Lima. Thadeu Luz and Felipe de Albuquerque which had a recent opportunity to speak at JSConf US and Campus Party.
```
Comment (hannesvdvreken) [inline _posts/2015-09-21-jsdaymcz-2015.md]: `well trustworthy` -> `well known`, or `trustworthy`.

"like Wilson Mendes from ThoughtWorks, Bruno Lima from CESAR, Thadeu Luz, and Felipe de Albuquerque. Felipe had a recent opportunity to speak at JSConf US and Campus Party."

Maybe add some links to this last sentence.

File: _posts/2015-09-21-jsdaymcz-2015.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+## Friday
+
+We had a pre-JSday meetup with drinks and good food for participants while we had a great time discussing JavaScript and Mobile Development with the Ionic SDK. We also launched the Ionic Alagoas Meetup with this pre-JSday.
+
+<blockquote class="twitter-tweet tw-align-center" lang="en"><p lang="en" dir="ltr">Our first <a href="https://twitter.com/Ionicframework">@Ionicframework</a> Alagoas Meetup the night before <a href="https://twitter.com/hashtag/JSday?src=hash">#JSday</a> 🙌 <a href="http://t.co/Hhh7Dv9zFZ">pic.twitter.com/Hhh7Dv9zFZ</a></p>&mdash; Juarez Filho (@juarezpaf) <a href="https://twitter.com/juarezpaf/status/642506115327086592">September 12, 2015</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Saturday
+
+Wilson Mendes’ talk was all about the future of AngularJS with the upcoming 2.0 version but first he went to a journey talking about the beginning of AngularjS and the huge popularity of JavaScript on Github. Showing the current 2.0 changes he blowed our minds with the [exciting bright future for AngularJS](http://www.slideshare.net/WilsonMendes/angular-js-20-46452230).
```
Comment (hannesvdvreken) [inline _posts/2015-09-21-jsdaymcz-2015.md]: `AngularjS` -> `AngularJS`

but first he went to a journey talking about -> but first he took us on a journey by talking about

Showing the current 2.0 changes he blowed our minds with the exciting bright future for AngularJS
->
He blew our minds with the 2.0 changes and the exciting bright future for AngularJS

File: _posts/2015-09-21-jsdaymcz-2015.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+You know you can do a lot with Google Maps API and in Thulio Philipe’s talk he explained how it’s easy to add Maps to your application and even how to customize its settings, a truly step by step how to get it working fast and to finish his talk he [created a sample](http://thulioph.github.io/twitter-realtime) using the Twitter API, Socket.io and of course Google Maps API.
```
Comment (hannesvdvreken) [inline _posts/2015-09-21-jsdaymcz-2015.md]: and even how to customize its settings, a truly step by step how to get it working fast and to finish his talk he

->

and even how to customize its settings..*\<end of sentence\>* **It was** a truly step by step **guide** on how to get it working fast.*\<end of sentence\>* **T**o finish his talk he created a sample **application**...

File: _posts/2015-09-21-jsdaymcz-2015.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+We had a lot of NodeJS topics inside of JSday and to start with this we had Thadeu Luz’s talk not only showing the basics of this amazing tool, but also a nice live coding guiding the attendees to setup it locally and after showing the NodeJS power.
```
Comment (hannesvdvreken) [inline _posts/2015-09-21-jsdaymcz-2015.md]: We had a lot of NodeJS topics **on** JSday.*\<end of sentence\>* **First was** Thadeu Luz’s talk showing the basics of this amazing tool, but also **featuring** a nice live coding **session that guided** the attendees to setup **NodeJS** locally.*\<end of sentence\>* After that he showed **off** the **power of** NodeJS.

File: _posts/2015-09-21-jsdaymcz-2015.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+Most JSday’s presentations showed a step by step or a quick getting started to the attendees and I’m sure it contributed a lot to awaken those which was expecting to learn more about JavaScript.
+
+Marina Limeira’s talk was about Firefox OS, resources, APIs you can use to build your apps, how you can speed up the development with Mozilla tools and of course how you can be part of this amazing community.
+My talk was about the Ionic SDK, it was a pleasure close JSday talking about Mobile Development using AngularJS, Cordova and the power of Ionic environment. I talked about how Ionic is empowering Web Developers with great tools and transforming them into Mobile Developers.
+
+It was an amazing experience have so much diversity in JSday and bring education to almost 200 attendees. Hope to see another JSday coming up soon.
```
Comment (hannesvdvreken) [inline _posts/2015-09-21-jsdaymcz-2015.md]: Hope to see -> We hope to see

File: _posts/2015-09-21-jsdaymcz-2015.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+JavaScript Robotics is a very exciting title and we didn’t expected anything less from Felipe de Albuquerque’s talk. He showed in a funny way how to bring JavaScript to the Robotics field and the crowed was perplexed when he controlled a Drone with JS. We also learned about the Nodebots, a term to describe the union between Hardware and JavaScript powered by NodeJS.
+
+Most JSday’s presentations showed a step by step or a quick getting started to the attendees and I’m sure it contributed a lot to awaken those which was expecting to learn more about JavaScript.
+
+Marina Limeira’s talk was about Firefox OS, resources, APIs you can use to build your apps, how you can speed up the development with Mozilla tools and of course how you can be part of this amazing community.
+My talk was about the Ionic SDK, it was a pleasure close JSday talking about Mobile Development using AngularJS, Cordova and the power of Ionic environment. I talked about how Ionic is empowering Web Developers with great tools and transforming them into Mobile Developers.
```
Comment (hannesvdvreken) [inline _posts/2015-09-21-jsdaymcz-2015.md]: (add an extra line break before this to separate paragraphs)

My talk was about the Ionic SDK.*\<end of sentence\>* **I**t was a pleasure **to** close JSday...

File: _posts/2015-09-21-jsdaymcz-2015.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+Meututor is a Startup in the education field aiming to create a nice environment for students learn about Math, Physics, Geography and more. In Khwesten Heiner’s presentation we had the opportunity to see how they builded the web app and also showed their mistakes and how they overpassed those errors to create a better solution to everyone.
+
+Hotaviano Melo’s talk was a journey where he talked about the last 3 years working inside of ZUQ, a loca software development company focused in urban mobility. The choices, the changes, the love and hate for some JavaScript solutions.
+
+JavaScript Robotics is a very exciting title and we didn’t expected anything less from Felipe de Albuquerque’s talk. He showed in a funny way how to bring JavaScript to the Robotics field and the crowed was perplexed when he controlled a Drone with JS. We also learned about the Nodebots, a term to describe the union between Hardware and JavaScript powered by NodeJS.
+
+Most JSday’s presentations showed a step by step or a quick getting started to the attendees and I’m sure it contributed a lot to awaken those which was expecting to learn more about JavaScript.
```
Comment (hannesvdvreken) [inline _posts/2015-09-21-jsdaymcz-2015.md]: Most of the JSday presentations showed a step by step guide or a quick getting started to the attendees.*\<end of sentence\>* I’m sure it woke up those who expected to learn more about JavaScript.

Comment (hannesvdvreken) [inline _posts/2015-09-21-jsdaymcz-2015.md]: Maybe move this paragraph to the bottom? Right after the paragraph on your talk.

**Comment by juarezpaf** [comment]
@hannesvdvreken could you please take a look again? :smile:

File: _posts/2015-09-23-jsdaymcz-2015.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Saturday
+
+![](/assets/posts/2015-09-23-jsdaymcz-2015/stage.jpg)
+
+[Wilson Mendes](https://twitter.com/willmendesneto)’ talk was all about the future of AngularJS with the upcoming 2.0 version but first he took us on a journey by talking about the beginning of AngularJS and the huge popularity of JavaScript on Github. He blew our minds with the 2.0 changes and the [exciting bright future for AngularJS](http://www.slideshare.net/WilsonMendes/angular-js-20-46452230).
+
+You know you can do a lot with Google Maps API and in [Thulio Philipe](http://twitter.com/thulioph_)’s talk he explained how it’s easy to add Maps to your application and even how to customize its settings. It was a truly step by step how to get it working fast. To finish his talk he [created a sample application](http://thulioph.github.io/twitter-realtime) using the Twitter API, Socket.io and of course Google Maps API.
```
Comment (hannesvdvreken) [inline _posts/2015-09-23-jsdaymcz-2015.md]: how it’s easy -> how easy it is

File: _posts/2015-09-23-jsdaymcz-2015.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+
+We had a lot of NodeJS topics on JSday. First was [Thadeu Luz](http://twitter.com/thadeuluz)’s talk showing the basics of this amazing tool, but also featuring a nice live coding session that guided the attendees to setup NodeJS locally. After that he showed off the power of NodeJS.
+
+[Bruno Lima](http://twitter.com/brunolimawd)’s talk started with a nice slide showing cute cats in a smartwatch with “Wearable JavaScript” title. He showed different ways to use JavaScript in real life to control or interact with a variety of devices, highlighted mentions goes to [Cylon.js](http://cylonjs.com/), [Johnny-Five](http://johnny-five.io) and [Pebble Watch](http://developer.getpebble.com/).
+
+[Electron](http://electron.atom.io) is formerly known as Atom Shell, developed by GitHub focused on build cross platform desktop apps with web technologies. The most powerful tool I know created with Electron is the amazing Slack Desktop app. In [Filipe Falcão](http://twitter.com/filipefalcaos)’s presentation he showed the basics and how to get started with this great tool powered also by NodeJS.
```
Comment (hannesvdvreken) [inline _posts/2015-09-23-jsdaymcz-2015.md]: [Electron](http://electron.atom.io), formerly known as Atom Shell, developed by GitHub, focuses on cross platform desktop apps with web technologies.

...and how to get started with this great NodeJS powered tool.

File: _posts/2015-09-23-jsdaymcz-2015.md [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+
+[Hotaviano Melo](http://twitter.com/melotechnology)’s talk was a journey where he talked about the last 3 years working inside of ZUQ, a loca software development company focused in urban mobility. The choices, the changes, the love and hate for some JavaScript solutions.
+
+JavaScript Robotics is a very exciting title and we didn’t expected anything less from [Felipe de Albuquerque](https://twitter.com/felipedeolinda)’s talk. He showed in a funny way how to bring JavaScript to the Robotics field and the crowed was perplexed when he controlled a Drone with JS. We also learned about the Nodebots, a term to describe the union between Hardware and JavaScript powered by NodeJS.
+
+[Marina Limeira](http://twitter.com/marinalimeira_)’s talk was about Firefox OS, resources, APIs you can use to build your apps, how you can speed up the development with Mozilla tools and of course how you can be part of this amazing community.
```
Comment (hannesvdvreken) [inline _posts/2015-09-23-jsdaymcz-2015.md]: [Marina Limeira](http://twitter.com/marinalimeira_)’s talk was about Firefox OS. The talk covered Firefox OS resources for developers, APIs you can use to build your apps, and how you can speed up the development with Mozilla tools. The talk also showed how you can be part of this amazing community.

File: _posts/2015-09-23-jsdaymcz-2015.md [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+
+[Bruno Lima](http://twitter.com/brunolimawd)’s talk started with a nice slide showing cute cats in a smartwatch with “Wearable JavaScript” title. He showed different ways to use JavaScript in real life to control or interact with a variety of devices, highlighted mentions goes to [Cylon.js](http://cylonjs.com/), [Johnny-Five](http://johnny-five.io) and [Pebble Watch](http://developer.getpebble.com/).
+
+[Electron](http://electron.atom.io) is formerly known as Atom Shell, developed by GitHub focused on build cross platform desktop apps with web technologies. The most powerful tool I know created with Electron is the amazing Slack Desktop app. In [Filipe Falcão](http://twitter.com/filipefalcaos)’s presentation he showed the basics and how to get started with this great tool powered also by NodeJS.
+
+[Juscyllan](http://twitter.com/kujhy)’s presentation brought to us a tool called [LoopBack](http://loopback.io/), the Node.js API Framework, showing how he created a mobile app using the LoopBack API capabilities, besides that also presented a short step by step on how to consume the API using AngularJS.
```
Comment (hannesvdvreken) [inline _posts/2015-09-23-jsdaymcz-2015.md]: [Juscyllan](http://twitter.com/kujhy)’s presentation brought to us **to** a tool called [LoopBack](http://loopback.io/), the Node.js API Framework.*\<end of sentence\>* **H**e created a mobile app using LoopBack API framework **and** besides that **he** also presented a short step by step guide on how to consume the API using AngularJS.

File: _posts/2015-09-23-jsdaymcz-2015.md [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+
+Meututor is a Startup in the education field aiming to create a nice environment for students learn about Math, Physics, Geography and more. In [Khwesten Heiner](http://github.com/Khwesten)’s presentation we had the opportunity to see how they builded the web app and also showed their mistakes and how they overpassed those errors to create a better solution to everyone.
```
Comment (hannesvdvreken) [inline _posts/2015-09-23-jsdaymcz-2015.md]: link to Meututor?

nice environment for students **to** learn about

Comment (hannesvdvreken) [inline _posts/2015-09-23-jsdaymcz-2015.md]: how they builded -> how they buil**t**

Comment (hannesvdvreken) [inline _posts/2015-09-23-jsdaymcz-2015.md]: the web app.*\<end of sentence\>*. Kwesten also showed **the** mistakes **they made** and how they overpassed those to create a better solution **for** everyone.

File: _posts/2015-09-23-jsdaymcz-2015.md [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+
+[Hotaviano Melo](http://twitter.com/melotechnology)’s talk was a journey where he talked about the last 3 years working inside of ZUQ, a loca software development company focused in urban mobility. The choices, the changes, the love and hate for some JavaScript solutions.
```
Comment (hannesvdvreken) [inline _posts/2015-09-23-jsdaymcz-2015.md]: [Hotaviano Melo](http://twitter.com/melotechnology)’s talk was a tale about his last 3 years working inside ZUQ, a loca**l** software development company. **ZUQ** focuse**s** on urban mobility. He talked about the choices, changes, and love & hate for some JavaScript solutions.

File: _posts/2015-09-23-jsdaymcz-2015.md [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+
+JavaScript Robotics is a very exciting title and we didn’t expected anything less from [Felipe de Albuquerque](https://twitter.com/felipedeolinda)’s talk. He showed in a funny way how to bring JavaScript to the Robotics field and the crowed was perplexed when he controlled a Drone with JS. We also learned about the Nodebots, a term to describe the union between Hardware and JavaScript powered by NodeJS.
```
Comment (hannesvdvreken) [inline _posts/2015-09-23-jsdaymcz-2015.md]: crowed -> crowd

Comment (hannesvdvreken) [inline _posts/2015-09-23-jsdaymcz-2015.md]: controlled a Drone with JS -> controlled a drone with JavaScript

**Comment by hannesvdvreken** [comment]
after this it's good to publish.

**Pull request merged**